core/src/test/scala/unit/kafka/log/LogCleanerManagerTest.scala

```diff
@@ -29,25 +29,142 @@ import org.junit.Assert._
 import org.junit.{After, Test}
 import org.scalatest.junit.JUnitSuite
 
+import scala.collection.immutable.HashSet
```

Comment:
unused import

Reply:
Hm... yeah. My IntelliJ isn't highlighting that for some reason